packages/blade/scripts/buildBlade.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# This script is used to build the blade package.
+set -e
+
+yarn build:clean
```

Comment:
why move it here? and not do run-s from the package.json?

Reply:
This is much easier to maintain and visualise the bundle steps, that having a `run-s build*` where the order of steps depend on how you arrange the scripts on package.json

Reply:
We can also use it without glob pattern like `run-s build:clean build:generate-types build:react-prod build:react-dev build:react-native`, right?

Reply:
> We can also use it without glob pattern

Yes, done that. 

That that it's super hard to look at. 
<img width="1174" alt="image" src="https://github.com/razorpay/blade/assets/35374649/38ef4cb0-601b-47ee-b67c-cb997f3f2e53">